src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function getSum(numb1) {
```

Comment:
you can manually return getSum function and delete return on 23 line

```suggestion
  return function getSum(numb1) {
```